src/ui/server/constants/integrity.ts

```diff
@@ -1,6 +1,6 @@
 export const INTEGRITY = {
   JS: 'sha512-BFOXngFJ+9msC1y5Q1LD6AG+LG635vp8Lfv/+HVCUU8w/N8VWH13wdTyZeLGvu4d7zoTz83nfz329ENYfe9x5Q==',
-  ACCESSIBILITY: 'sha512-+7iHxg+GAStsu3dNSm9llhRRnjOvuU10+1sbYjZlkCUd5wH9iaCWV9fY+33BtUQHafrjWU/rM42x3QlAj68Z2g==',
+  ACCESSIBILITY: 'sha512-rXOeAno+kqPr6+BmVOd1b+7ZNBcuaO6l8P2qczcUzrfUBUY/bHqiiLnKJahGYqJWZDis1xHuH9Uai3iTjM+MoA==',
```

Comment:
Can you check the other integrity hashes, looks like the GA/GTM integrity hash will be broken (and prevent the script from loading), since it's been changed

Reply:
was not getting any errors for the other hashes, and the tests seem to be passing

Reply:
I would suggest running `npm run build` to ensure the assets files are correct since `css-minimizer-webpack-plugin` has been added. Please ignore if already done.

Reply:
Is there a need to run `npx prisma generate` at `api`?